integration_tests/dbt_project.yml

```diff
@@ -1,13 +1,12 @@
 name: 'marketo_integration_tests'
-version: '0.7.0'
+version: '0.8.0'
 profile: 'integration_tests'
 config-version: 2
 
 vars:
-
-    marketo__enable_programs: True
-    marketo__enable_campaigns: True
-
+    # marketo__activity_delete_lead_enabled:  false     # Un-comment line to disable testing the activity_delete_lead table 
+    marketo__enable_campaigns:   true      # Un-comment line to enable testing the campaigns table
+    marketo__enable_programs:    true      # Un-comment line to enable testing the programs table
```

Comment:
I have removed the commented lines but tested locally with it uncommented. I need to include lines 8 and 9 or they won't get included in the documentation since the default is false in this case.